### ISSUE TITLE
Field Collection is not filled during installation

### ISSUE THREAD

**Issue by aKugler** [basilicom/pimcore-data-quality-bundle#12]
Hi!

During installation, the field collection is only created but not filled with fields. I did not see any errors during the installation process. Error logs also show nothing. However, if I understand correctly, the structure of the import JSON has changed. This may be the reason.
Tested with Pim 11.1.5 (Admin 1.3.4) and 11.2.1 (1.4.0)

DataQualityFieldDefinition.php:
```
<?php

/**
 * Fields Summary:
 */

return \Pimcore\Model\DataObject\Fieldcollection\Definition::__set_state(array(
   'dao' => NULL,
   'key' => 'DataQualityFieldDefinition',
   'parentClass' => '',
   'implementsInterfaces' => '',
   'title' => 'dataQualityConfig.text.title',
   'group' => '',
   'layoutDefinitions' => 
  \Pimcore\Model\DataObject\ClassDefinition\Layout\Panel::__set_state(array(
     'name' => NULL,
     'type' => NULL,
     'region' => NULL,
     'title' => NULL,
     'width' => 0,
     'height' => 0,
     'collapsible' => false,
     'collapsed' => false,
     'bodyStyle' => NULL,
     'datatype' => 'layout',
     'children' => 
    array (
    ),
     'locked' => false,
     'blockedVarsForExport' => 
    array (
    ),
     'fieldtype' => 'panel',
     'layout' => NULL,
     'border' => false,
     'icon' => NULL,
     'labelWidth' => 100,
     'labelAlign' => 'left',
  )),
   'fieldDefinitionsCache' => NULL,
   'blockedVarsForExport' => 
  array (
  ),
));

```
Best regards
Andreas

**Comment by abhiramalli** [comment]
@aKugler 
have you fixed this issue

**Comment by aKugler** [comment]
@abhiramalli 
No, I didn't do that. At the time, I was just evaluating various plugins to see if they could be used for our purposes.  For testing purposes, I only added the fields from the file ([https://github.com/basilicom/pimcore-data-quality-bundle/blob/main/src/Resources/install/fieldcollection_sources/fieldcollection_DataQualityFieldDefinition_export.json](https://github.com/basilicom/pimcore-data-quality-bundle/blob/main/src/Resources/install/fieldcollection_sources/fieldcollection_DataQualityFieldDefinition_export.json)) using the Admin UI.

**Comment by fmcsecurity** [comment]
@aKugler 
Thanks for sharing the advice! 

Same hiccup on a clean install of v11.2.3, all's well after manually creating the fields.

**Comment by hoangngx** [comment]
SOLVED!!!

Use this file instead and it should work like a charm

[fieldcollection_DataQualityFieldDefinition_export.json](https://github.com/user-attachments/files/15585905/fieldcollection_DataQualityFieldDefinition_export.json)
